_LowLevelCode/cpp/combine_sqw/combine_sqw.cpp

```diff
@@ -1,8 +1,8 @@
-#include "../CommonCode.h"
-
+#include "CommonCode.h"

```

Comment:
`CommonCode.h` is still in the top `cpp` folder. Is that now on the path so this include will find the file correctly?

Reply:
I've put an `include_directories()` command in `_LowLevelCode/cpp/CMakeLists.txt` so every C++ target will now include the `cpp` directory.